rotkehlchen/chain/evm/decoding/ens/name_utils.py

```diff
@@ -0,0 +1,18 @@
+import re
+from typing import Final
+
+TLD_PATTERN: Final = re.compile(r'\.[a-z0-9-]{2,}$', re.IGNORECASE)
+
+
+def is_potential_ens_name(text: str) -> bool:
+    """Function used to tell if a string could be a potential ENS domain.
+    If .eth is found we exit fast but for other domains like .argent.xyz
+    we use a regex that detects valid TLDs. The developer has always to check
+    if indeed the address is valid or not.
+
+    The full list of TLDs supported by ENS is available at https://docs.ens.domains/dns/tlds
+    """
```

Comment:
Simplification of the text. Also question. What do you mean by "The developer has to check
    if indeed the address is valid or not."

```suggestion
    """Function used to tell if a string could be a potential ENS domain.
    We use a regex that detects valid TLDs. The developer has to check
    if indeed the address is valid or not.

    The full list of TLDs supported by ENS is available at https://docs.ens.domains/dns/tlds
    """
```

Reply:
I meant that we don't try to resolve it so it might be that the TLD is not supported. But I believe is already clear by the name of the function

Reply:
> But I believe is already clear by the name of the function

yes, so you can remove it